src/javascripts/components/mobile-navigation-section.mjs

```diff
@@ -110,16 +110,53 @@ class MobileNavigationSection extends Component {
       if (this.expanded) {
         this.$subnav.removeAttribute('hidden')
       }
-      this.$button.removeAttribute('hidden')
+      removeAttributes(this.$button, Object.keys(attributesForHidingButton))
     } else {
       this.$root.parentElement.classList.remove(
         'app-mobile-navigation-section__service-navigation-item'
       )
-      this.$button.setAttribute('hidden', '')
+      setAttributes(this.$button, attributesForHidingButton)
       this.$subnav.setAttribute('hidden', '')
       this.$root.removeAttribute('hidden')
     }
   }
 }
 
+/**
+ * Collection of attributes that needs setting on a `<button>`
+ * to fully hide it, both visually and from screen-readers,
+ * and prevent its activation while hidden
+ */
+const attributesForHidingButton = {
```

Comment:
I wonder if there is a point in making this more global and generic? I assume every element that uses `hidden` would benefit from this?